CHANGELOG.md

```diff
@@ -6,10 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added 
+- solr maxBooleanClauses increased from 1024 til 8192. This is because licensmodule queries can have several 100 terms. (solr.config=1.7.2)
+  For this to also work on a solr cloud setup the solr start script must have:  export SOLR_OPTS="-Dsolr.allowPaths=/solr-c-backup -Dsolr.max.booleanClauses=8192
```

Comment:
Could and should you add this to the cloud_start.sh script in the bin folder of the project, which contains a solr cloud setup?

Reply:
Added to the custom cloud_start.sh script